suitcase/spec.py

```diff
@@ -799,7 +1009,7 @@ def _get_motor_name(start):
     motor_name = start['motors']
     # We only support a single scanning motor right now.
     if len(motor_name) > 1:
-        warnings.warn(
+        raise NotImplementedError(
```

Comment:
(Not covered line)
`.format()` is being passed a tuple incorrectly. This should be
```python
"position".format(len(motor_name), motor_name))
```